gql/resolver.go

```diff
@@ -372,6 +372,14 @@ func (dr *dealResolver) IsVerified() bool {
 	return dr.ProviderDealState.ClientDealProposal.Proposal.VerifiedDeal
 }
 
+func (dr *dealResolver) KeepUnsealedCopy() bool {
```

Comment:
I think you may also need to add these fields to `gql/schema.graphql` and to `react/src/gql.js`